components/dev-tools/cmake/SPECS/cmake.spec

```diff
@@ -0,0 +1,94 @@
+#----------------------------------------------------------------------------bh-
+# This RPM .spec file is part of the OpenHPC project.
+#
+# It may have been modified from the default version supplied by the underlying
+# release package (if available) in order to apply patches, perform customized
+# build/install configurations, and supply additional files to support
+# desired integration conventions.
+#
+#----------------------------------------------------------------------------eh-
+
+%include %{_sourcedir}/OHPC_macros
+
+%define pname cmake
+
+%define major_version 3.8
+%define minor_version 1
+
+Summary: CMake is an open-source, cross-platform family of tools designed to build, test and package software.
+Name: %{pname}%{PROJ_DELIM}
+Version: %{major_version}.%{minor_version}
+Release: %{_rel}%{?dist}
+# https://spdx.org/licenses/BSD-3-Clause.html
+License: BSD-3-Clause
+Group: %{PROJ_NAME}/dev-tools
+URL: https://cmake.org/
+Source: https://cmake.org/files/v%{major_version}/cmake-%{version}.tar.gz
+BuildRequires:  gcc-c++
+BuildRequires:  ncurses-devel
+BuildRequires:  curl-devel
+BuildRequires:  xz-devel
+BuildRequires:  zlib-devel
+BuildRequires:  pkgconfig
+
+%define install_path %{OHPC_PUB}/%{pname}/%version
```

Comment:
Not sure where the cmake in the OpenHPC package hierarchy belongs, but I would recommend not a top level directory but rather %{OHPC_PUB}/apps/%{pname}/%version . I see that other packages are also installed directly in the top level pub directory, so it makes kind of sense if looking at the other packages but apps still seems nicer as a top-level directory.